lib/groonga-query-log/command/run-regression-test.rb

```diff
@@ -426,6 +428,20 @@ def create_option_parser
                   "(#{@notifier_options[:mail_only_on_failure]})") do |boolean|
           @notifier_options[:mail_only_on_failure] = boolean
         end
+        parser.on("--[no-]verify-cancel",
+                  "Verify cancellation",
+                  "(#{@verify_cancel})") do |boolean|
+          @verify_cancel = boolean
+        end
+        parser.on("--cancel-max-wait=SECONDS", Float,
+                  "Used with `--verify_cancel`." +
```

Comment:
We can't use Markdown:

```suggestion
                  "Used with --verify_cancel. " +
```